### ISSUE TITLE
Error: Android cannot find symbol Leanplum.setAplicationContext(this);

### ISSUE THREAD

**Issue by AugustoAleGon** [Leanplum/Leanplum-ReactNative-SDK#68]
After following the Leanplum documentation on Android https://docs.leanplum.com/reference/react-native-setup
And also trying the autolink and the manual linking and both didn't work.

I still got this:
```
MainApplication.java:89: error: cannot find symbol
    Leanplum.setAplicationContext(this);
            ^
  symbol:   method setAplicationContext(MainApplication)
  location: class Leanplum
```

**Comment by augusto-booment** [comment]
@hborisoff I already did that. But I got the same issue mention above. Any idea? Autolink is also broken too.

**Comment by hborisoff** [comment]
@augusto-booment 
I'm sending you my [sample project](https://drive.google.com/file/d/1ixjRG3PfuzuBKpwgHLpGfKQ3R-It-lGO/view?usp=sharing) I created few minutes ago.
This is not the full Leanplum integration, but only to the point where you cannot compile it.

These are the commands executed:

1. react-native init NewProject
2. npm install
2. npm install @leanplum/react-native-sdk
3. npx react-native link @leanplum/react-native-sdk
4. Added the following lines in android/app/build.gradle:

```
implementation project(':@leanplum_react-native-sdk')
implementation "com.leanplum:leanplum-fcm:${LEANPLUM_SDK_VERSION}"
implementation 'com.google.firebase:firebase-messaging:20.1.4'
```

5. Added the following lines in onCreate method of MainApplication.java:

```
// Leanplum setup
Leanplum.setApplicationContext(this);
Parser.parseVariables(this);
//  For session lifecyle tracking.
LeanplumActivityHelper.enableLifecycleCallbacks(this);
```

6. Run it with "yarn android" and it starts on the device
